### DISCUSSION TITLE
Looking for the v2 equivalent of /api/v1/system/version/upgrade to check for pfSense upgrades available?

### DISCUSSION THREAD

**Discussion by mbentley** [jaredhendrickson13/pfsense-api#539] [Q&A] [answered]
I looked through the Swagger documentation but I didn't see an equivalent v2 version of `/api/v1/system/version/upgrade`.  I am also open to other ideas, even things outside of this project to find the latest version available of pfsense CE programmatically.  Seems like there is a lack of RSS feeds, published release APIs, etc for pfsense.

**Comment by jaredhendrickson13** [comment, marked as answer]
That endpoint is being reworked a bit. There were some odd discrepancies in how it works between CE and Plus so it didn't make it into the initial v2 release. Might be in v2.1.0 but I wouldn't count on that. You can install v1 alongside v2 and continue to use the v1 endpoint in the meantime.

Another workaround for CE you could do with v2 is just run the command below using /api/v2/diagnostics/command_prompt. It simply checks if there's a system upgrade available in the upstream repo:

```
pfSense-upgrade -c
```
